### DISCUSSION TITLE
Copy Tiddler to Clipboard Button

### DISCUSSION THREAD

**Discussion by GameDungeon** [TiddlyWiki/TiddlyWiki5#7687] [Ideas]
Now that tiddlers are copy & pastable I was planning to make a pr that adds a copy tiddler to clipboard button. I'm wondering, would this be better as a separate button, or should it be bundled in with the export button?

**Comment by Jermolene** [comment]
Hi @GameDungeon that's an excellent idea.

I favour a separate button/menu item "Copy tiddler" that defaults to using the JSON export template. Copying in multiple formats could be useful in some situations, but there's a clear use case for rapidly copying and pasting tiddlers between wikis.

**Comment by GameDungeon** [comment]
That's what I was thinking. A copy button that loads both a tiddler (as you can store a tiddler in the clipboard) and the json format as a fallback.

**Reply by Jermolene** [reply]
Exactly, I'd welcome a PR along those lines thank you @GameDungeon

**Reply by GameDungeon** [reply]
@Jermolene How can I convert a tiddler to the json format? I can't seem to find the code that does it.

Edit: I think I figured it out, I should use renderTiddler
